wgpu-core/src/instance.rs

```diff
@@ -382,25 +406,19 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
         &self,
         desc: &RequestAdapterOptions,
         unsafe_extensions: wgt::UnsafeExtensions,
-        inputs: AdapterInputs<Input<G, AdapterId>>,
     ) -> Option<AdapterId> {
         let instance = &self.instance;
         let mut token = Token::root();
         let (surface_guard, mut token) = self.surfaces.read(&mut token);
         let compatible_surface = desc.compatible_surface.map(|id| &surface_guard[id]);
         let mut device_types = Vec::new();
 
-        let id_vulkan = inputs.find(Backend::Vulkan);
-        let id_metal = inputs.find(Backend::Metal);
-        let id_dx12 = inputs.find(Backend::Dx12);
-        let id_dx11 = inputs.find(Backend::Dx11);
```

Comment:
Because the backends are already pre-selected, I don't create these ids anymore...

Reply:
Got the feedback that I can just leave in the backend selection here as a second filter step, that also provides the method with ids, so I will do that.